non-wheel/active-jobs-dashboard/root.tpl

```diff
@@ -0,0 +1,64 @@
+<html>
+<head><title>Active Job Dashboard</title>
+<script type="text/javascript" src="jquery-latest.js"></script>
```

Comment:
I have not seen scripts included with the "integrity" and "crossorigin" attributes before, but trust your more-able insight that this is secure. :)